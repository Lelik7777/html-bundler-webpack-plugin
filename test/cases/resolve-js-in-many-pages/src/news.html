<!DOCTYPE html>
<html>
<head>
  <title>News</title>
  <script src="./common.js" defer="defer"></script>
  <script src="./b.js" defer="defer"></script>
</head>
<body>
  <h1>News</h1>
</body>
</html>