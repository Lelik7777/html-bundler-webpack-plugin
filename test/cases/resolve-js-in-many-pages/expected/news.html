<!DOCTYPE html>
<html>
<head>
  <title>News</title>
  <script src="js/common.64204441.js" defer="defer"></script>
  <script src="js/b.26d1b659.js" defer="defer"></script>
</head>
<body>
  <h1>News</h1>
</body>
</html>